scimath/Fitting/test/tLSQFit.cc

```diff
@@ -352,8 +352,8 @@ int main() {
       val1f[0] = 1;
       for (uInt j1=1; j1<6; j1++) val1f[j1] = val1f[j1-1]*511;
       lsq5.makeNorm(val1f, 1.0f, val12f[511]);
-      MemoryIO memio;
-      AipsIO aio(&memio);
+      std::shared_ptr<ByteIO> memio(new MemoryIO());
```

Comment:
```suggestion
      std::shared_ptr<MemoryIO> memio(new MemoryIO());
```
Additionally, you could use `std::make_shared` here.